docs/integration-guides/key-management.md

```diff
@@ -329,16 +329,91 @@ curl -d '{
     As a result of the command above, the nano\_node will return a signed, but not yet broadcasted transaction. Broadcasting of the signed transaction is covered in the [Broadcasting Transactions](#broadcasting-transactions) section.
 
 !!! info "Manually receiving first block"
-    The very first transaction on an account-chain, which is always a receive, is a bit special since it doesn't have a `"previous"` block. The process however, is very similar to a conventional receive transaction.
+    The very first transaction on an account-chain, which is always a receive, is slightly special and deserves its own section [First Receive Transaction](#first-receive-transaction).
 
-    | Field          | Description |
-    |                |             |
-    | previous       | Value is 0 (32 0's) |
-    | account        | Same as normal receive. |
-    | representative | Choose a reliable, trustworthy representative. |
-    | balance        | Same as normal receive. This will be the transaction amount of the pairing send. |
-    | link           | Same as normal receive. |
-    | key            | Same as normal receive. |
+---
+
+#### First Receive Transaction
+
+The first transaction of every account is crafted in a slightly different way. To open an account, you must have sent some funds to it with a [Send Transaction](#send-transaction) from another account. The funds will be **pending** on the receiving account. If you already know the hash of the pending transaction, you can skip Step 1.
```

Comment:
Change to "The first transaction of an account is crafted"...saying "every" might seem like it changes for each account.

Reply:
Changed